### PR TITLE
Add 'first session' example in the README

### PR DESCRIPTION
**Pull request by trufae**
**Checklist**

- [ ] Closing issues: #issue
- [ ] Mark this if you consider it ready to merge
- [ ] I've added tests (optional)
- [ ] I wrote some lines in the [radare2book](https://github.com/radareorg/radare2book)

**Description**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,6 +37,21 @@ r2env init
...
 ```
 
+## Usage
+
+These are the first steps to use r2, read the book or find tutorials for more details
+
+```sh
+$ r2 /bin/ls   # open the binary in read-only mode
+> px 32        # print 32 byte hexdump current block
+> f~foo        # filter flags with ~grep (same as |grep)
```
Comment (alex-bender) [inline README.md:47]: Having that it's for  the first steps, it's better to introduce jargon slowly, so better show example on function list, for example
```
$ r2 /bin/ls
> aaa           # analyze functions and cross references, same as `r2 -A`
> afl~sym    # filter functions with `~` ...

> oo+;w hello    # write `hello` in current offset
```
would be better to introduce `seek` in that list

Comment (trufae) [reply README.md:47]: chk again, feel free to suggest more changes

Comment (alex-bender) [reply README.md:50]: looks good. Actually I'm thinking about some tutorial being in code. Kind of vimtutor

**Comment by trufae** [comment]
Merging then! :) feel free to submit another PR improving it if you notice anything

**Pull request merged**